TransformationSystem/Client/TaskManagerPlugin.py

```diff
@@ -106,16 +106,18 @@ def _ByJobType( self ):
     destSites = set( res['Value'] )
 
     # 2. get JobTypeMapping "Exclude" value (and add autoAddedSites)
+    gLogger.debug( "Getting JobTypeMapping 'Exclude' value (and add autoAddedSites)" )
     jobType = self.params['JobType']
     if not jobType:
       raise RuntimeError( "No jobType specified" )
     excludedSites = self.opsH.getValue( 'JobTypeMapping/%s/Exclude' % jobType, '' )
+    gLogger.debug( "Excluded sites for %s jobs: %s" % ( jobType, excludedSites ) )
     excludedSites = excludedSites + ',' + ','.join( fromChar( self.opsH.getValue( 'JobTypeMapping/AutoAddedSites', '' ) ) )
 
     # 3. removing sites in Exclude
     if not excludedSites:
       pass
-    elif excludedSites == 'ALL' or 'ALL' in excludedSites:
+    elif excludedSites == 'ALL' or 'ALL,' in excludedSites:
```

Comment:
'ALL' in excludedSites.split(',') ?
And btw, in the opsH, I believe that if you give list() as default type, it will automatically do a split like that.